src/main/java/com/zontik/groshiky/controllers/IndexController.java

```diff
@@ -55,8 +56,12 @@ public String addUser(@ModelAttribute("user") User user, Model model) {
     }
 
     @RequestMapping(value = "/dashboard", method = RequestMethod.GET)
-    public String dashboard(ModelMap model) {
+    public String dashboard(Model model) {
+        User user = userService.findUserById(((User) SecurityContextHolder.getContext().getAuthentication().getPrincipal()).getId());
```

Comment:
It's better to move this logic to separate method. I think you'll need this in a future (maybe some Util fille, or parent Controller, like BaseController)